content/applications/finance/documents.rst

```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
      rights to the document. It is possible to replace the owner of a document. I.e.: an employee
      must be owner of a document to be able to see it in "My Profile".
 
-Finally, different :guilabel:`Actions` are available at the bottom of the right panel, depending on
-the workspace where your document is stored.
+Finally, different :ref:`Actions <documents/workflow-actions>` are available at the bottom of the
```

Comment:
Different :ref:`Actions <documents/workflow-actions>` are available at the bottom of the